plugins/chat/assets/javascripts/discourse/lib/chat-message-interactor.js

```diff
@@ -249,8 +250,61 @@ export default class ChatMessageInteractor {
   }
 
   @action
-  markAsActive(state) {
+  markAsActive(state, event) {
```

Comment:
@jjaffeux maybe you want to tweak this, I just moved the old hovering logic into here since when you hovered on the message actions on desktop you could not click on anything, it would just flicker since we were just hovering out straight away and setting the activeMessage to null. Works nicer now, in the thread and the main section